### ISSUE TITLE
The bottom of the letter tiles are vibrating

### ISSUE THREAD

**Issue by tom7** [jcreedcmu/wordlike#163]
I invited the letters to Sit, but their bottom sides vibrate:

https://github.com/jcreedcmu/wordlike/assets/499244/9da2c245-18c5-449b-8297-da771ba2fe23

I ask that the letters do not vibrate. 

> [!IMPORTANT]
> The vibration increases my stress and heart pressure.

**Comment by jcreedcmu** [comment]
The vibration ensures that dust and particulate matter does not accumulate on the tiles

**Comment by jcreedcmu** [comment]
Regarding your P0 escalation reason, to wit
> [!IMPORTANT]
> The vibration increases my stress and heart pressure.
Karolina points out "How do you think the letters feel??"

**Comment by jcreedcmu** [comment]
But seriously, hmm. I'm not entirely sure how to reproduce or diagnose yet. I'm reasonably confident I'm making the same WebGL call to draw the rack tiles every frame, and I'm not yet sure why that would produce a nondeterministic result.

**Comment by jcreedcmu** [comment]
I notice [this stackoverflow post](https://stackoverflow.com/questions/49974467/webgl-flickering-in-chrome-on-windows-x64-with-nvidia-gpu) happens to some observe weird behavior on just the very combination of browser, OS, and gpu that I think you have? 

(a) Do the letters in letter-game also flicker on firefox on your machine?
(b) Do you get any weird behavior at all in the [small repro](https://sergeyext.github.io/sergeyext/flicker.html) linked to by the above stackoverflow post?

**Comment by tom7** [comment]
No, that repo looks normal to me on my Chrome. No firefox on this machine (sorry I just reinstalled everything) but it doesn't happen in Edge.
I think it's because I'm zoomed in to 150% by default on chrome. This also causes some oddities, I just noticed like being able to see a sliver of pixels of the map at the top of the DEATH TIMER.

Also: If you zoom in and out while the game is running, it gets really messed up!
